src/Database/Expression/WindowExpression.php

```diff
@@ -149,12 +248,45 @@ public function sql(ValueBinder $generator): string
             $partitionSql = 'PARTITION BY ' . implode(', ', $expressions);
         }
 
-        return sprintf(
+        $sql = sprintf(
             'OVER (%s%s%s)',
             $partitionSql,
             $partitionSql && $this->order ? ' ' : '',
             $this->order ? $this->order->sql($generator) : ''
         );
+
+        if ($this->frame) {
```

Comment:
The frame bound needs to be placed inside of the `OVER` clause.

Reply:
That is true lol. Looks like I kept following one mistyped test case.

Fortunately, there will be several annoying db query tests implemented for this later.